source/request_response.c

```diff
@@ -317,6 +317,24 @@ struct aws_http_stream *aws_http_stream_new_client_request(const struct aws_http
     return stream;
 }
 
+int aws_http_stream_configure_server_request_handler(
+    struct aws_http_stream *stream,
+    const struct aws_http_request_handler_options *options) {
+    if (!options || options->self_size == 0 || !options->server_connection) {
+        AWS_LOGF_ERROR(
+            AWS_LS_HTTP_CONNECTION,
+            "id=%p: Cannot create client request, options are invalid.",
+            (void *)(options ? options->server_connection : NULL));
+        aws_raise_error(AWS_ERROR_INVALID_ARGUMENT);
+        return AWS_OP_ERR;
```

Comment:
`aws_raise_error` returns `AWS_OP_ERR`, so you can just write: `return aws_raise_error(AWS_ERROR_INVALID_ARGUMENT);`